README.md

```diff
@@ -6,36 +6,114 @@
 [![codecov](https://codecov.io/gh/ZcashFoundation/zebra/branch/main/graph/badge.svg)](https://codecov.io/gh/ZcashFoundation/zebra)
 ![License](https://img.shields.io/badge/license-MIT%2FApache--2.0-blue.svg)
 
-### 🚧 UNDER CONSTRUCTION 🚧
+## About
 
 [Zebra](https://zebra.zfnd.org/) is the Zcash Foundation's independent,
 consensus-compatible implementation of the Zcash protocol, currently under
 development.  Please [join us on Discord](https://discord.gg/na6QZNd) if you'd
 like to find out more or get involved!
 
+## Alpha Releases
+
+Our first alpha release is planned for December 8th, 2020.
+
+The goals of this release are to:
+- participate in the Zcash network,
+- replicate the Zcash chain state, and
+- implement the Zcash proof of work consensus rules.
+
```

Comment:
Maybe we should add something here about what Zebra doesn't do?
```suggestion
- implement the Zcash proof of work consensus rules,
- sync on Mainnet under excellent network conditions.

Currently, Zebra does not validate all the Zcash consensus rules. It may be
unreliable on Testnet, and under less-than-perfect network conditions. See
our [current features](#current-features) and [roadmap](future-work) for
details.
```